### PR TITLE
Use string column for in_business

### PR DESCRIPTION
**Pull request by jonallured**
This commit also includes a temporary rake task to migrate existing data on production.

### CODE REVIEW

File: lib/tasks/temporary/in_business_data_migration.rake [hunk 1]
```diff
@@ -0,0 +1,15 @@
+desc 'Covert in_business to a string'
```
Comment (gnilekaw) [inline lib/tasks/temporary/in_business_data_migration.rake]: This task is in the temporary directory. Are you planning on deleting it? Or is `temporary` a signal that these tasks were run in the past?

File: lib/tasks/redshift.rake [hunk 2]
```diff
@@ -26,7 +26,7 @@ namespace :redshift do
           city character varying, \
           country character varying, \
           email character varying, \
-          in_business boolean, \
+          in_business character varying, \
```
Comment (gnilekaw) [inline lib/tasks/redshift.rake]: 👍 

We end up changing the schema often. ... makes me wonder when we should consider automating the drop/create task when it changes.

**Comment by gnilekaw** [comment]
Looks good!

File: lib/tasks/temporary/in_business_data_migration.rake (hunk 1 above)
Comment (jonallured) [inline lib/tasks/temporary/in_business_data_migration.rake]: Yeah, once I've run these data migrations, I'm going to have a PR that removes them. They'll always be in the git history, but I don't want them lingering around.

File: lib/tasks/redshift.rake (hunk 2 above)
Comment (jonallured) [inline lib/tasks/redshift.rake]: I have had a similar thought, but didn't do anything about it. :\

**Pull request merged**